bolt/include/bolt/Rewrite/DWARFRewriter.h

```diff
@@ -89,6 +89,15 @@ class DWARFRewriter {
   /// Store Rangelists writer for each DWO CU.
   RangeListsDWOWriers RangeListsWritersByCU;
 
+  /// Contains ranges writer and die.
```

Comment:
Briefly describe what this datatype is used for. The reader could see the contents in the lines below.